src/components/_video/VideoPlayer/VideoOverlays/EndingOverlay.tsx

```diff
@@ -103,17 +103,32 @@ export const EndingOverlay: React.FC<EndingOverlayProps> = ({
   }, [currentThumbnailUrl, randomNextVideo, randomNextVideoThumbnailUrl])
 
   return (
-    <OverlayBackground>
+    <OverlayBackground
+      onClick={(e) => {
```

Comment:
Let's move this function to const. Inline functions creates new reference on each render which cause unnecessary rerender.

Reply:
Moving it to const doesn't really change anything, `const` will have a new reference on each render as well

Reply:
Yeah I was thinking about `useCallback`. But either way, we have 2 inline functions here which does the same, so maybe it's worth to declare it above?

Reply:
Sure let's move it to `const` if we duplicate but let's not overuse `useCallback` if not necessary